src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -422,7 +423,10 @@ def get_object_data(
             urls_map[o.variable_key].append(o.document_url)
 
         for key, variable in state.variables.items():
-            submission_value = dynamic_values[key]
+            try:
+                submission_value = dynamic_values[key]
+            except PathAccessError:
+                continue
```

Comment:
This solution isn't pretty but the current structures/utilities in the code does not allow to do so